modules/_labs/color-picker/react/lib/parts/ColorSets.ts

```diff
@@ -0,0 +1,95 @@
+import {colors} from '@workday/canvas-kit-react-core';
+
+export const defaultColorSet = [
+  colors.cinnamon400,
+  colors.peach400,
+  colors.chiliMango400,
+  colors.cantaloupe400,
+  colors.sourLemon400,
+  colors.juicyPear400,
+  colors.kiwi400,
+  colors.greenApple400,
```

Comment:
It's a little weird that the relationship between rows and columns is implicit (as shown here by newline spacing). I think the final API should have a stronger contract between rows and columns.

Reply:
agreed. We should address this before moving it out of labs 👍